OWNERS

```diff
@@ -1,18 +1,10 @@
 approvers:
```

Comment:
This should be org owners/governing board, IMO.

Reply:
For many repos, that would be sensible, but for the website, many standard changes, including updating redirects and applying security updates require a top-level approver given how the folders are required to be structured.